### PR TITLE
[2999][ADD] product_website_description

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[2999](https://www.quartile.co/web?#id=2999&action=771&model=project.task&view_type=form&menu_id=505)
- Add new tab "Product Description for Website Display" on Product Master
- Add fields Website Description field into the tab
- Website Description field be able to be edited with HTML

### CODE REVIEW

File: website_product_description_view/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Product Description View",
+    "version": "15.0.1.0.0",
+    "category": "website",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "depends": ["base", "website"],
```
Comment (yostashiro) [inline website_product_description_view/__manifest__.py]:
```suggestion
    "depends": ["website_sale"],
```

File: website_product_description_view/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    website_description = fields.Html(
+        "Description for the website",
+        render_engine="qweb",
+        sanitize_attributes=False,
+        translate=html_translate,
+        sanitize_form=False,
+    )
+    can_write = fields.Boolean(
+        compute="_compute_can_write", help="The current user can edit the template."
+    )
```
Comment (yostashiro) [inline website_product_description_view/models/product_template.py]: What is the purpose of these changes? We do not need this file as far as I am aware.

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: website_product_description_view/models/product_template.py (hunk 2 above)
Comment (rinaldifirdaus) [reply website_product_description_view/models/product_template.py]: I set the module to depends to base & website, thus when i didn't add this file, Odoo cannot find the website_description. but i just tried to change the dependencies to website_sale, there is no issue without this file. 
I will revise and add the improvement soon. Thank you!

File: website_product_description_view/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
+This module does the following:
+
+- Add new tab "Product Description for Website Display" on Product Master
+- Add fields Website Description field into the tab
+- Website Description field be able to be edited with HTML
```
Comment (yostashiro) [inline website_product_description_view/readme/DESCRIPTION.rst]:
```suggestion
- Adds "Product Description for Website Display" tab to the product form,
  to give users the ability to edit the product description to show on the
  eCommerce product page.

This feature is in response to the situation where it was not possible to edit
the product description on the shop frontend for a technical glitch.
```

**Review by yostashiro — COMMENTED** [review]
Can we rename the module to product_website_description, please.

**Review by yostashiro — APPROVED** [review]
LGTM 👍
